### ISSUE TITLE
Ep3: Moving to the current folder - use different file extensions?

### ISSUE THREAD

**Issue by gcapes** [swcarpentry/shell-novice#924]
This exercise is a thought experiment, but it isn't really clear why the two files are in the wrong directory. They're all `*ose.dat` files. If the file ending or name were to indicate the file had been processed it might be a little easier to follow, or least be better motivated.

I'd also like the exercise to be runnable.

What do you think about the above suggestions?

**Comment by gdevenyi** [comment]
I agree this question starts out a bit vague. It feels like it could use more intro.

If we want it to be runnable it will have to be redone against the standard files we're supplying, or create the files in-line.

I'm +1 for the fixing up the motivation, but I think the point of this lesson is picking out individual files one-by-one. I think adding a suffix might make this lesson a bit too simple.

I'm -1 on runnable as it either makes the setup too complicated, or adds to our existing example data.

**Comment by colinmorris** [comment]
Agree that the premise/motivation here is unclear.

Another problem I'm seeing with this exercise is that it's the first (and only) time they're seeing `mv` used with more than two arguments. I don't think it's reasonable to expect them to know the semantics of `mv a b c`. 

This pattern is introduced better for `cp` in the exercise "Copy with Multiple Filenames".

"Moving to the Current Folder" has enough problems that I would support just nuking it (and maybe adding a brief note to the solution of "Copy with Multiple Filenames" mentioning that the same rules apply to `mv`).
